indigo/files.py

```diff
@@ -309,3 +312,42 @@ def _put_file_in_zip_file(zipfiles, file_name_in_storage, file_name_in_zip):
             data = fp.read()
             for zipfile in zipfiles:
                 zipfile.writestr(file_name_in_zip, data)
+
+
+def update_data_quality_report_file_for_all_projects():
+
+    # Data
+    data = {"single_fields": []}
+
+    fields = [
+        i
+        for i in settings.JSONDATAFERRET_TYPE_INFORMATION["project"]["fields"]
+        if i.get("type") != "list"
+        # Because we only generate stats on public data anyway, running stats on the status field makes no sense.
+        and i.get("key") != "/status"
```

Comment:
This is made redundant by the line below.

Reply:
I realised that technically but left for readability, tho I probably should have just tried to deal with that with comments